install/0000_80_machine-config-operator_01_machineconfig.crd.yaml

```
@@ -340,7 +340,7 @@ spec:
                 nullable: true
               kernelType:
                 description: Contains which kernel we want to be running like default
-                  (traditional), realtime
+                  (traditional), realtime, 64k pages
```

(traditional), realtime, 64k-pages
```suggestion
                  (traditional), realtime, 64k pages
```